app/gui/view/component-browser/component-group/src/new_entry.rs

```diff
@@ -0,0 +1,454 @@
+//! A module containing definition for Component Browser Entry [`View`] and related structures.
+
+use crate::prelude::*;
+
+use crate::icon;
+use crate::new_entry::style::Colors;
+use crate::set::GroupId;
+use crate::set::SectionId;
+
+use enso_frp as frp;
+use ensogl_core::application::command::FrpNetworkProvider;
+use ensogl_core::application::frp::API;
+use ensogl_core::application::Application;
+use ensogl_core::data::color;
+use ensogl_core::display;
+use ensogl_core::display::scene::Layer;
+use ensogl_core::display::Scene;
+use ensogl_grid_view as grid_view;
+use ensogl_grid_view::entry::Contour;
+use ensogl_grid_view::entry::MovedHeaderPosition;
+use ensogl_hardcoded_theme::application::component_browser::component_group as theme;
+use ensogl_shadow as shadow;
+use ensogl_text as text;
+
+
+// ==============
+// === Export ===
+// ==============
+
+pub mod style;
+
+pub use crate::new_entry::style::Style;
+
+
+
+// =================
+// === Constants ===
+// =================
+
+/// The number of pixels the entries inside one group overlap each other.
+///
+/// The entries need to overlap, otherwise we see artifacts at their boundaries.
+const ENTRIES_OVERLAP_PX: f32 = 1.0;
+/// The padding between entry background and declared entry contour.
+///
+/// Cannot be `0.0`, because the background is clipped more sharply than the selection shape, making
+/// selection extending a bit over the background what looks bad.
```

Comment:
```suggestion
/// Cannot be `0.0` because the background is clipped more tightly than the selection shape, which means that the selection extends over the background. This looks bad.
```